packages/react-icons/convert.js

```diff
@@ -134,14 +134,10 @@ function processFolder(srcPath, destPath, resizable) {
       var jsxCode = resizable ? svgr.default.sync(iconContent, svgrOpts, { filePath: file }) : svgr.default.sync(iconContent, svgrOptsSizedIcons, { filePath: file })
       var jsCode = 
 `
-const ${destFilename}Icon = (iconProps: FluentIconsProps) => {
-  const { className, primaryFill } = iconProps;
-  return ${jsxCode};
-}
 
-export const ${destFilename} = /*#__PURE__*/wrapIcon(/*#__PURE__*/${destFilename}Icon({}), '${destFilename}');
-      `
 
+export const ${destFilename} = /*#__PURE__*/wrapIcon(/*#__PURE__*/${jsxCode}, '${destFilename}');
```

Comment:
With the removal here of iconProps, does that impact the way that anything like the aria or other custom properties might be applied to the asset?

Reply:
No, those props will still be applied in the `wrapIcon` method. They were unnecessary here, and actually increased bundle size without needing to